### ISSUE TITLE
Add label to social links

### ISSUE THREAD

**Issue by rshipp** [BanchanArt/banchan#550]
**Issue**: Links must have discernable text
**Impact**: Serious
**Recommendation**: Add an `aria-label` to these links, so the screenreader can announce where they lead. Additionally, links that use `target=”_blank”` should be used minimally, and should announce that they open in a new tab. E.g. `aria-label=”Banchan on Mastodon (opens in new tab)”`.
**More info**: https://www.w3.org/WAI/WCAG21/Techniques/aria/ARIA8, https://www.w3.org/WAI/WCAG21/Techniques/general/G200, https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#external_links_and_linking_to_non-html_resources 
**Screenshot**:
![image](https://github.com/BanchanArt/banchan/assets/1382374/56b5cf1e-21ce-4152-afb3-e1079d3a3885)
